src/WebJobs.Script/functionTemplate.js

```diff
@@ -1,9 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-var f = require('{0}');
+var f = require('{0}'),
+    util = require('util');
 
 return function (context, callback) {{
+    var origLog = context.log;
+    context.log = function(value) {{
+        origLog(util.inspect(value));
```

Comment:
if I'm reading this right, then isn't util.inspect always called, even when the arg is a string? I think that might be problematic because util.inspect will wrap a string in single quotes e.g.

> console.log(util.inspect('test'))
'test'